include/Basic/TokenKind.def

```diff
@@ -102,13 +107,34 @@ PUNCTUATOR(comma, ',')
 PUNCTUATOR(colon, ':')
 PUNCTUATOR(semi, ';')
 PUNCTUATOR(arrow, '->')
-PUNCTUATOR(ellipsis, '...')
 PUNCTUATOR(equal, '=')
 PUNCTUATOR(backslash, '\\')
-PUNCTUATOR(exclamation, '!')
 PUNCTUATOR(question, '?')
-PUNCTUATOR(stringQuote, '\"')
-PUNCTUATOR(singleQuote, '\'')
+PUNCTUATOR(at, '@')
+
+// Operators
+OPERATOR(plus, '+')
+OPERATOR(sub, '-')
+OPERATOR(mul, '*')
+OPERATOR(div, '/')
+OPERATOR(mod, '%')
+OPERATOR(eq, '==')
+OPERATOR(ne, '!=')
+OPERATOR(lt, '<')
+OPERATOR(le, '<=')
+OPERATOR(gt, '>')
+OPERATOR(ge, '>=')
+OPERATOR(and, '&&')
+OPERATOR(or, '||')
+OPERATOR(bitAnd, '&')
+OPERATOR(bitOr, '|')
+OPERATOR(bitXor, '^')
+OPERATOR(bitLShift, '<<')
+OPERATOR(bitRShift, '>>')
+OPERATOR(range, '...')
+OPERATOR(exclusiveRange, '..<')
+OPERATOR(exclamation, '!')
+OPERATOR(tilde, '~')
```

Comment:
why do all the operators have names based on their usage, but these two have names based on their symbol

not, compl